dmoj/executors/NODEJS.py

```diff
@@ -0,0 +1,44 @@
+from dmoj.executors.script_executor import ScriptExecutor
+from dmoj.cptbox.filesystem_policies import ExactFile
+
+
+class Executor(ScriptExecutor):
+    ext = 'js'
+    command = 'node'
+    nproc = -1
+    command_paths = ['node', 'nodejs']
+    syscalls = [
+        # Start syscalls copied from COFFEE.py
+        'capget',
+        'eventfd2',
+        'newselect',
+        'sched_yield',
+        'select',
+        'setrlimit',
+        'statx',
+        # End syscalls copied from COFFEE.py
```

Comment:
Can we try trimming down this list? Also the comment is not necessary.

Reply:
Thanks for the suggestion! I ran `/judge/.docker/entry test | grep NODEJS` multiple times while commenting out these, and it seems like we only need `capget` and `eventfd2`. I left these two (and `shutdown`) in.